### ISSUE TITLE
UI: IP address column always empty on the Add VM modal

### ISSUE THREAD

**Issue by OlegChuev** [apache/cloudstack#8273]
The modal form uses a field that is absent in the API response.

This modal makes an API request to the 'listVirtualMachines' API endpoint and prints all VMs. In the `ui/src/views/network/PortForwarding.vue` file at line `428`, the column expects to use the `ip` field, but this field is absent in the actual response. I assume it should be `publicip` instead of `ip`. Please feel free to close the issue if I'm missing something.

If it should be just `publicip` instead of `ip`, let me know, and I can prepare a fix.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.18
~~~

##### STEPS TO REPRODUCE

~~~
1) Login to the ACS
2) Create VPC
3) Create VM with Network created on the previous step
4) Navigate to the VPC -> Public IP Addresses tab
5) Click on the non-"source nat" public ip address
6) Navigate to the Port Forwarding tab 
7) Click "Add" button
8) Check VM table
~~~

<img width="605" alt="Screenshot 2023-11-27 at 19 12 16" src="https://github.com/apache/cloudstack/assets/91337563/b347a1ad-6dfb-455f-9ad8-14bdd5c24c7b">

<img width="827" alt="Screenshot 2023-11-27 at 19 11 50" src="https://github.com/apache/cloudstack/assets/91337563/a7878725-f3a3-4fea-82db-fe30177d9169">

<img width="854" alt="Screenshot 2023-11-27 at 19 01 34" src="https://github.com/apache/cloudstack/assets/91337563/1fce9411-f363-416b-81c3-1fe74531bf29">

##### EXPECTED RESULTS

~~~
IP Address value is present 
~~~

##### ACTUAL RESULTS

~~~
IP Address value is missing 
~~~

**Comment by OlegChuev** [comment]
hi @weizhouapache, should I prepare the fix and remove this column from the modal at all?

**Comment by weizhouapache** [comment]
> hi @weizhouapache, should I prepare the fix and remove this column from the modal at all?

Do you agree with removing the field ?
anyone who is interested, including you, can create a PR for it. If you want, go ahead, otherwise I can create a PR, it is very simple.

**Comment by weizhouapache** [comment]
fixed via #8275 
thanks @OlegChuev
